distribution/docs/src/main/jdocs/content/_plugins/event-processor-contents.adoc

```diff
@@ -0,0 +1,77 @@
+:type: plugin
+:status: published
+:title: Event Processor
+:link: _event_processor
+:plugintypes: postingest
+:summary: Creates, updates, and deletes subscriptions.
+
+The Event Processor provides an engine that creates, updates, and deletes subscriptions for event notification.
+These subscriptions optionally specify a filter criteria so that only events of interest to the subscriber are posted for notification.
+
+An internal subscription tracker monitors the OSGi registry, looking for subscriptions to be added (or deleted).
```

Comment:
❓ I guess it depends on whether it is monitoring them before they are added or after?